server/routes/manageConditions/handlers/outofboundsPremises/outOfBoundsPremisesInputRoutes.ts

```diff
@@ -0,0 +1,40 @@
+import { Request, Response } from 'express'
+import LicenceService from '../../../../services/licenceService'
+
+export default class OutOfBoundsPremisesInputRoutes {
+  constructor(private readonly licenceService: LicenceService) {}
+
+  POST = async (req: Request, res: Response): Promise<void> => {
+    const { licenceId } = req.params
+    const { conditionId } = req.params
+    const { user, licence } = res.locals
+
+    const { code } = licence.additionalLicenceConditions.find(c => c.id === parseInt(conditionId, 10))
+
+    let redirect = `/licence/create/id/${licenceId}/additional-licence-conditions/condition/${code}/outofbounds-premises`
+    if (req.query?.fromPolicyReview) {
+      redirect += '?fromPolicyReview=true'
```

Comment:
does this policy review stuff not work similar to MEZ? 
(no probs keeping it in here for now)

Reply:
it should do I think. I just copied this from MEZ so should be exactly the same